src/topology/bases.lean

```diff
@@ -317,6 +322,79 @@ lemma _root_.set.pairwise_disjoint.countable_of_nonempty_interior [separable_spa
   countable a :=
 (h.mono $ λ i, interior_subset).countable_of_is_open (λ i hi, is_open_interior) ha
 
+/-- A set in a topological space is separable if it is contained in the closure of a
+countable set.-/
+def is_separable (s : set α) :=
+∃ c : set α, countable c ∧ s ⊆ closure c
```

Comment:
This is a weird definition. Does it exist in informal math? It feels really weird that this isn't equivalent to `separable_space (subtype s)`.